plugins/templates/README.md

```diff
@@ -0,0 +1,75 @@
+# Templates Plugin
+> Hapi templates plugin for the Screwdriver API
+
+## Usage
+
+### Register plugin
+
+```javascript
+const Hapi = require('hapi');
+const server = new Hapi.Server();
+const templatesPlugin = require('./');
+
+server.connection({ port: 3000 });
+
+server.register({
+    register: templatesPlugin,
+    options: {}
+}, () => {
+    server.start((err) => {
+        if (err) {
+            throw err;
+        }
+        console.log('Server running at:', server.info.uri);
+    });
+});
+```
+
+### Routes
+
+#### Get all templates
+`page` and `count` optional
+
+`GET /templates?page={pageNumber}&count={countNumber}`
+
+#### Get single template
+
+`GET /templates/{id}`
```

Comment:
For builds and jobs, the `id` is a number. What's the `id` for a template: is also a number, or is it the name of the template+version number?